pubsublite/internal/wire/streams.go

```diff
@@ -210,13 +220,10 @@ func (rs *retryableStream) connectStream() {
 		rs.mu.Lock()
 		defer rs.mu.Unlock()
 
-		if rs.status == streamReconnecting {
+		if rs.status == streamReconnecting || rs.status == streamTerminated {
```

Comment:
Nit: the comment probably needs updating if you merge the two conditionals.

Reply:
I just reverted this to keep the conditionals separate.